packages/itwinui-css/backstop/tests/information-panel.html

```diff
@@ -1361,13 +1361,17 @@ <h1>Information panel</h1>
                 onclick="switchInfoPanelOrientation(this);"
               >
                 <!-- Docked right -->
-                <svg
+                <span
                   class="iui-button-icon"
-                  viewBox="0 0 16 16"
-                  xmlns="http://www.w3.org/2000/svg"
+                  aria-hidden="true"
                 >
-                  <path d="m0 2v12h16v-12zm11 11h-10v-10h10z" />
-                </svg>
+                  <svg
+                    viewBox="0 0 16 16"
+                    xmlns="http://www.w3.org/2000/svg"
+                  >
+                    <path d="m0 2v12h16v-12zm15 7h-14v-6h14z" />
```

Comment:
I changed the path from [L1369](https://github.com/iTwin/iTwinUI/pull/788/files#diff-697a9ad7c898ef8688dea9c11a68c149c5d000f54c7113193a6e94d1faaaf9ccL1369) since I found a bug: The icon first shows the dock right icon. Then after clicking it, it still shows the dock right icon. After clicking it again, it shows the dock bottom icon.

Now, the icon starts off with the dock bottom icon.

Bug screenshot:

[Video1.webm](https://user-images.githubusercontent.com/45748283/198397265-7c26e779-b2e1-44f3-8584-c0ceac01448c.webm)